Tests/Common/TestUtils.cs

```diff
@@ -74,9 +74,14 @@ public static void AssertFileDoesNotExist(string fileName, string parentDirector
             Assert.IsFalse(File.Exists(fullPath), "Not expecting file to exist: {0}", fullPath);
         }
 
-        public static string CreateTextFile(string fileName, string directory, string content = null)
+        public static string CreateTextFile(string relativeFileName, string directory, string content = null)
         {
-            string fullPath = Path.Combine(directory, fileName);
+            string fullPath = Path.Combine(directory, relativeFileName);
+
+            // Ensure the directory exists
+            string fullDirectory = Path.GetDirectoryName(fullPath);
+            Directory.CreateDirectory(fullDirectory);
+
```

Comment:
Minor tweak to the helper function so it can be passed relative file names e.g.
  CreateTextFile("foo\\file.txt", "c:\\dir") 
will create "c:\\dir\foo\\file.txt".